### PR TITLE
Save rep name, witness, contentions, periods, evidence, comments

### PR DESCRIPTION
**Pull request by sharonwarner**
Connects #2815 

- [x] Fill out rep name, witness, contentions, periods, evidence, comments
- [x] Refresh page
- [x] Confirm fields are still filled out

### CODE REVIEW

File: client/app/hearings/actions/Dockets.js [hunk 1]
```diff
@@ -117,3 +118,25 @@ export const onCommentsForAttorneyChange = (commentsForAttorney) => ({
...
+
+  ApiUtil.patch(`/hearings/worksheets/${worksheet.id}`, { data: { worksheet } }).
+  then(() => {
+    dispatch({ type: Constants.SET_WORKSHEET_EDITED_FLAG_TO_FALSE });
+  },
+  () => {
+    dispatch({ type: Constants.SET_WORKSHEET_SAVE_FAILED,
+      payload: { saveFailed: true } });
+  });
+  dispatch({ type: Constants.TOGGLE_WORKSHEET_SAVING });
```
Comment (NickHeiner) [inline client/app/hearings/actions/Dockets.js]: Why would we want to fire this again after firing the `patch` but before it returns?

File: client/app/hearings/HearingWorksheet.jsx [hunk 2]
```diff
@@ -113,7 +121,6 @@ export class HearingWorksheet extends React.PureComponent {
             <label htmlFor="worksheet-contentions">Contentions</label>
             <Textarea
               name="Contentions"
-              minRows={3}
```
Comment (nolawi) [inline client/app/hearings/HearingWorksheet.jsx]: weird this was not removed but git is showing as removed

**Comment by nolawi** [comment]
Ran locally, all 6 fields tested saved! LGTM

File: client/app/hearings/HearingWorksheet.jsx (hunk 2 above)
Comment (sharonwarner) [reply client/app/hearings/HearingWorksheet.jsx]: Yeah, we had it listed twice.

Comment (nolawi) [reply client/app/hearings/HearingWorksheet.jsx]: 👍

File: app/controllers/hearings/worksheets_controller.rb [hunk 3]
```diff
@@ -10,13 +10,23 @@ def show
     end
   end
 
-  # def update
-  #   worksheet.update!(worksheet_params)
-  #   render json: { worksheet: hearing_worksheet }
-  # end
+  def update
+    worksheet.update!(worksheet_params)
+    worksheet.class.repository.update_vacols_hearing!(worksheet.vacols_record, worksheet_params)
```
Comment (aroltsch) [inline app/controllers/hearings/worksheets_controller.rb]: You don't have to do that here. In the Hearing model, `update` is overwritten and calls `update_vacols_hearing!`

Comment (sharonwarner) [reply app/controllers/hearings/worksheets_controller.rb]: Oh awesome! Updating now...

**Pull request merged**